ieee754r/fabsd32.c

```diff
@@ -40,19 +37,13 @@
 DEC_TYPE
 INTERNAL_FUNCTION_NAME (DEC_TYPE x)
 {
-  decContext context;
-  decNumber dn_result;
-  DEC_TYPE result;
-  decNumber dn_x;
-
-  FUNC_CONVERT_TO_DN (&x, &dn_x);
-
-  decContextDefault (&context, DEFAULT_CONTEXT);
-  decNumberAbs (&dn_result, &dn_x, &context);
-
-  FUNC_CONVERT_FROM_DN (&dn_result, &result, &context);
-
-  return result;
+  DEC_TYPE result = x;
+  unsigned char *p = &result;
+  #if __BYTE_ORDER == __LITTLE_ENDIAN
+    p += _DECIMAL_SIZE/8 -1;
```

Comment:
While not wrong, I would add a note and compiler check to ensure BID or DPD is used. Also minor nit, a space between - and 1.

Reply:
Also, you could probably getaway with p += sizeof(DEC_TYPE) - 1

Reply:
>I would add a note and compiler check to ensure BID or DPD is used

How can I do that? :baby: 
I've saw only how to check about BID: `#ifdef __DECIMAL_BID_FORMAT__` 
Other thing is, if we are not using neither DPD or BID what is the desired behavior, stop compiling?

Reply:
Ah, yes. I don't think there is an explicit test for DPD format.